rasterio/rio/warp.py

```diff
@@ -0,0 +1,197 @@
+import logging
+from math import ceil
+import click
+from cligj import format_opt
+
+from . import options
+import rasterio
+from rasterio import crs
+from rasterio.transform import Affine
+from rasterio.warp import (reproject, RESAMPLING, calculate_default_transform,
+   transform_bounds)
+
+
+logger = logging.getLogger('rio')
+
+
+@click.command(short_help='Warp a raster dataset.')
+@options.file_in_arg
+@options.file_out_arg
+@format_opt
+@options.like_file_opt
+@click.option('--dst-crs', default=None,
+              help='Target coordinate reference system.  Default: EPSG:4326')
```

Comment:
@brendan-ward The help says this defaults to `EPSG:4326` but the click default is not set.